config/main.py

```diff
@@ -1667,6 +1673,65 @@ def load_port_config(config_db, port_config_path):
                 port_name), display_cmd=True)
     return
 
+
+def load_golden_config(golden_config_path):
+    if not os.path.isfile(golden_config_path):
+        return
+
+    # Override configDB with golden config
+    clicommon.run_command('config override-config-table {}'.format(
+        golden_config_path), display_cmd=True)
+    return
+
+
+#
+# 'override-config-table' command ('config override-config-table ...')
+#
+@config.command('override-config-table')
+@click.argument('golden-config-db', required=True)
+@click.option(
+    '--dry_run', type=click.STRING,
+    help="Dry run, writes config to the given file"
+)
+@clicommon.pass_db
+def override_config_table(db, golden_config_db, dry_run):
+    """Override current configDB with golden config."""
+
+    try:
+        # Load golden config json
+        golden_config_input = read_json_file(golden_config_db)
+    except Exception:
```

Comment:
>Exception [](http://example.com/codeflow?start=11&length=9)

Use specific exception types, general one will hide bugs.

Reply:
Add print for exception
```
click.secho("Bad format: json file broken. {}".format(str(e))
```